library/src/main/java/YamlStringToSpecTreeConverter.java

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.LinkedHashMap;
+import org.yaml.snakeyaml.Yaml;
```

Comment:
Not familiar with Snake Yaml, how did we decide on this?

Reply:
snakeyaml is actually provided by Jackson! It provides a nice way to deserialize a YAML file into the spec tree (LinkedHashMap)